.gitlab-ci.yml

```diff
@@ -33,88 +90,112 @@ setup-vms:
 #
 # NOTE GitLab is set to remove images after a month, so they will be forced to
 # rebuild at least once a month.
-build-ws-base:
+build-dependencies:
   stage: build-deploy-base
   variables:
-    IMAGE_TAG: "dlsw/datafed/ws-base"
+    IMAGE_TAG: "dlsw/datafed/dependencies"
+    CACHE_TAG: "dlsw/datafed/cache:dependencies"
     GIT_STRATEGY: clone
   tags:
-    - ci-datafed-core
+    - ci-datafed
     - docker
```

Comment:
I don't think there is a runner with a ci-datafed tag. I think you can just use the docker tag and it will use what every runner is available.